### ISSUE TITLE
Folders not loading in 3.9.13 on Linux

### ISSUE THREAD

**Issue by makarandh** [standardnotes/desktop#805]
Seeing this issue after auto updating from 3.9.10 to 3.9.13.

Folders don't load. The left most pane says "There is an issue loading folders" and shows a reload button. Clicking on it doesn't help. Restarting doesn't help. 

Tried version 3.9.12 and seeing the same issue. 

Downloaded 3.9.10 again, and it's working in this version. 

### Issue details: 

**Environment:** Debian 11 (bullseye)
**Standard Notes version:** 3.9.13 and 3.9.12

Attached: Screenshot of the left pane of version 3.9.13

<img width="191" alt="Screenshot_20220106_074525" src="https://user-images.githubusercontent.com/38956972/148349351-5b6c3e54-3c2d-4154-b2b8-2d80fed93f0a.png">

**Comment by jbeldia** [comment]
Hi @makarandh, I’m sorry to hear about this issue, I’ve tried to resync your subscription with your account, so can I ask you to try signing out and signing back into v3.9.13 and let me know if it helped?

**Comment by jrabensc** [comment]
Had the same issue. Fixed it by disabling and enabling the folder toggle in the quick settings menu.

**Comment by kslstn** [comment]
Same problem after updating to 3.9.14 on macOS. Solution above still worked!

**Comment by moughxyz** [comment]
This issue is fixed in 3.9.15.

**Comment by makarandh** [comment]
Unfortunately I still see the issue. I still have to turn off folders and then turn back on again for folders to work. I tried signing out, clearing cache, and signing back again but it's still the same. When  I start Standard Notes after closing it, I have to turn off and turn on folders.

**Comment by khlinuxdeveloper** [comment]
@moughxyz the error **continues in linux version 3.9.20**.

Attached video: https://user-images.githubusercontent.com/68512577/150526459-9fb53acc-6a0a-48e2-ab7d-f7f02893075d.mp4

The only solution to load the Folders extension correctly is the following: https://github.com/standardnotes/desktop/issues/809#issuecomment-1011988477

_Version in use: 3.9.20 Linux-x86_64 (AppImage)
Operating System: Ubuntu 20.04_

**Comment by makarandh** [comment]
This is fixed for me in 3.10.0. The folder extension is gone. I migrated to the new nested tag system where the . (dot) separator for tags has been changed to / (slash) separators. So they act like folders. 

Most if it worked well however there were a little bit of problem here and there. 

1. 8 of the notes didn't get migrated to new system and still had the . separators for tags. I had to manually moved them. I have like thousands of notes, so I am glad I didn't see this issue in more notes. 

2. I had many folders that had the dot (.) in its name (not the tag). And subfolders were created in the new system even though they weren't present. I do have a lot of notes (hundreds) that were moved to such subfolders. I am dreading having to manually correct this problem. 


But anyway, the folder not loading issue is fixed. So I am closing it.
